qa/rpc-tests/wallet.py

```diff
@@ -119,6 +141,7 @@ def run_test (self):
         assert_equal(self.nodes[0].getbalance(), Decimal('19.99900000'))
 
         # Sendmany 10 BTC
```

Comment:
I'm not sure either.  I had put all those prints in there to help me 
debug and find issue 6554.  When you run
the python scripts they do help you to understand where the problem is 
happening.  I'd like to leave them
in for now but I was thinking that we need to update and standardize the 
logging of messages, maybe in the
next pull after this one. I'm working on getting everything into python, 
maybe we can update the logging then?



On 16/08/2015 5:23 AM, MarcoFalke wrote:
>
> In qa/rpc-tests/wallet.py 
> <https://github.com/bitcoin/bitcoin/pull/6548#discussion_r37145277>:
>
> > @@ -119,6 +141,7 @@ def run_test (self):
> >          assert_equal(self.nodes[0].getbalance(), Decimal('19.99900000'))
> >
> >          # Sendmany 10 BTC
>
> I am not sure which is preferred, but the code should only have either 
> a comment or the |print|, imo.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/bitcoin/bitcoin/pull/6548/files#r37145277>.
>